src/core/thread/mle_constants.hpp

```diff
@@ -79,8 +79,9 @@ enum
     kMinTimeoutDataPoll             = (OPENTHREAD_CONFIG_MINIMUM_POLL_PERIOD +
                            OPENTHREAD_CONFIG_FAILED_CHILD_TRANSMISSIONS * OPENTHREAD_CONFIG_RETX_POLL_PERIOD) /
                           1000, ///< Minimum timeout(s) for data poll
-    kMinTimeout = (kMinTimeoutKeepAlive >= kMinTimeoutDataPoll ? kMinTimeoutKeepAlive
+    kMinTimeout         = (kMinTimeoutKeepAlive >= kMinTimeoutDataPoll ? kMinTimeoutKeepAlive
                                                                : kMinTimeoutDataPoll), ///< Minimum timeout(s)
+    kChildTimeoutMargin = 10, ///< Margin for duplicate parent request
```

Comment:
How about renaming it to be something like`kParentRequestDuplicateMargin` or`kParentRequestDupMargin`?